bridges/snowbridge/primitives/core/src/outbound.rs

```diff
@@ -69,12 +69,16 @@ mod v1 {
 	#[derive(Clone, Encode, Decode, RuntimeDebug, TypeInfo)]
 	#[cfg_attr(feature = "std", derive(PartialEq))]
 	pub enum Command {
-		/// Execute a sub-command within an agent for a consensus system in Polkadot
-		AgentExecute {
-			/// The ID of the agent
+		/// Transfer ERC20 tokens
+		TransferToken {
```

Comment:
We can't remove `AgentExecute` unless we introduce `v2::Command` and ensure that outbound-queue does a just-in-time migration when reading 'v1::Command` messages from MessageQueue.

As with xcm version migrations, we need to be careful about ABI compatibility. The same goes for Solidity code in the gateway. We don't have the luxury of being able to halt the bridge and empty all the queues before upgrading it.

Reply:
https://github.com/Snowfork/snowbridge/pull/1155#discussion_r1560463955